### ISSUE TITLE
Color reset output when terminal does not support color

### ISSUE THREAD

**Issue by rmgk** [outr/scribe#144]
Starting with version 3.1.0 scribe outputs color reset codes (i.e., `[0`) at the beginning of each line, even when the output does not support them. I assume this to be because of this line which does not check the output type:

https://github.com/outr/scribe/blob/a0cd61593e3c7e9dacbb0fb8351018c0bd8c6bce/core/shared/src/main/scala/scribe/writer/ANSIConsoleWriter.scala#L34

**Comment by darkfrog26** [comment]
@rmgk can you clarify what you mean? If you're using the `ANSIConsoleWriter`, it assumes that ANSI codes are supported.

**Comment by darkfrog26** [comment]
FYI, I removed auto-detection of support because it was causing difficulties on certain environments.

**Comment by rmgk** [comment]
Sorry, I linked to and talked about different versions.
In Scribe 3.1.7, if I understand correctly, there is only a FileWriter and a ConsoleWriter, and to set the output format to no colors I would instead set the ASCIIOutputFormat as such:

```
        import scribe.format._
        val myFormatter: Formatter = formatter"$message ($position)"
        Logger.root.clearHandlers().withHandler(
          formatter = myFormatter,
          minimumLevel = Some(scribe.Level.Info),
          outputFormat = ASCIIOutputFormat
        ).replace()
```

However, in the line below (same as I linked before, but now in a different place) the color reset code is appended independently from the output format, so I get color codes at the beginning of each line in my log file. 

https://github.com/outr/scribe/blob/3.1.7/core/shared/src/main/scala/scribe/writer/SystemOutputWriter.scala#L33

**Comment by darkfrog26** [comment]
Oops, yeah, *slaps forehead*, that was a bonehead mistake on my part. I'll get it updated and a new release out ASAP.

Good catch!

**Comment by darkfrog26** [comment]
Released `3.1.8` with this fix
